packages/experiments/src/components/PersonaCoin/PersonaCoin.types.ts

```diff
@@ -1,12 +1,14 @@
-import { ImageLoadState } from 'office-ui-fabric-react';
-import { IComponent, IComponentStyles, IHTMLDivSlot, ISlotProp, IStyleableComponentProps } from '../../Foundation';
+import { ImageLoadState, IBaseProps } from 'office-ui-fabric-react';
+import { IComponentStyles, IHTMLDivSlot, ISlotProp, IComponent, IStyleableComponentProps } from '../../Foundation';
 import { IPersonaPresenceSlot } from '../../utilities/factoryComponents.types';
 import { IPersonaCoinImageSlot } from './PersonaCoinImage/PersonaCoinImage.types';
 
-export type IPersonaCoinComponent = IComponent<IPersonaCoinProps, IPersonaCoinTokens, IPersonaCoinStyles>;
+export type IPersonaCoinComponent = IComponent<IPersonaCoinProps, IPersonaCoinTokens, IPersonaCoinStyles, IPersonaCoinViewProps>;
 
 export type IPersonaCoinSlot = ISlotProp<IPersonaCoinProps> | string;
 
+export type IPersonaCoinSize10 = ISlotProp<{}>;
```

Comment:
Just a nit at this point, but I think you could move this to `PersonaCoinSize10.tsx`, define `PersonaCoinSizeProps` as `{}`, and then import it here. I think it'll improve maintainability a bit.

Reply:
(Also append `Slot` to the naming.)

Reply:
Sounds good :)